### PR TITLE
Updated README and .gitignore

### PR DESCRIPTION
**Pull request by oliver-nowak**
- Made the README a bit more clear.
- Updated the .gitignore

Mainly this is a test of merging Team-3 code back into Team-2.

### CODE REVIEW

**Comment by oliver-nowak** [comment]
@nkavanagh plz review

File: group1/comm/static/comm_tool.js [hunk 1]
```diff
@@ -291,7 +291,7 @@ function add_new_room(room) {
 function populate_user_list() {
   $.getJSON('http://' + server_host + ':' + server_port + '/api/users/?format=json', function(all_users) { 
 
-    $.getJSON('http://' + server_host + ':3000/users', function(connected_users){
+    $.getJSON('http://' + host_name + ':3000/users', function(connected_users){
```
Comment (nkavanagh) [inline group1/comm/static/comm_tool.js:294]: Where does this get set?

Comment (oliver-nowak) [reply group1/comm/static/comm_tool.js:294]: @nkavanagh good catch. it looks like i missed this when i merged your changes in.

**Comment by oliver-nowak** [comment]
@nkavanagh plz review new changes

**Comment by nkavanagh** [comment]
LGTM :shipit:

**Pull request merged**